samples/.NET/NUnit/Tests.cs

```diff
@@ -8,13 +8,11 @@ namespace PlaywrightTests;
 
 [Parallelizable(ParallelScope.Self)]
 [TestFixture]
-public class Tests : PageTest
+public class Tests : PageTestWithArtifact
```

Comment:
Is this a class created by us? Does a user always have to add this to use the service?